examples/sound/Sound.cpp

```diff
@@ -26,17 +25,24 @@ void playSound()
 
     // Create a sound instance and play it
     sf::Sound sound(buffer);
-    sound.play();
 
-    // Loop while the sound is playing
-    while (sound.getStatus() == sf::Sound::Playing)
+    for (int i = 0; i < 2; ++i)
```

Comment:
Maybe change "2" to some constant like "NUM_TIMES_SOUND_PLAYED"?
But I'm also not sure why we need to play it 2 times...

Reply:
The first loop the audio is played without the effect applied, then the second loop the audio is played with the effect, to demonstrate the difference in output.

Reply:
I think it would be good to have a visual indicator for this. E.g. show "No effect" for the first play and "Chorus Effect" for the second. Ideally the code would be something like this:

```cpp
playWithoutEffect(...);
playWithEffect(...);
```